pilot.go

```diff
@@ -88,10 +89,21 @@ func (c *chanController) OpenChannel(target *btcec.PublicKey,
 	// TODO(halseth): make configurable?
 	minHtlc := lnwire.NewMSatFromSatoshis(1)
 
-	updateStream, errChan := c.server.OpenChannel(
-		target, amt, 0, minHtlc, feePerKw, false, 0,
-	)
+	// Construct the open channel request and send it to the server to begin
+	// the funding workflow.
+	req := &openChanReq{
+		targetPubkey:    target,
+		chainHash:       *activeNetParams.GenesisHash,
+		localFundingAmt: amt,
+		pushAmt:         0,
+		minHtlc:         minHtlc,
+		fundingFeePerKw: feePerKw,
+		private:         false,
+		remoteCsvDelay:  0,
+		minConfs:        c.minConfs,
```

Comment:
The `WalletBalance` method here should be modified to pass in the min confs. Otherwise, it always asks for those that have one conf still.

Reply:
Fixed!